ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -236,12 +236,33 @@ spec:
           workspace: results
           subPath: summary
 
+    - name: resolve-pr-type
+      taskRef:
+        name: resolve-pr-type
+        kind: Task
+      runAfter:
+        - read-config
+      params:
+        - name: bundle_path
+          value: "$(tasks.detect-changes.results.bundle_path)"
+        - name: affected_catalogs
+          value: "$(tasks.detect-changes.results.affected_catalogs)"
+        - name: affected_catalog_operators
+          value: "$(tasks.detect-changes.results.affected_catalog_operators)"
+        - name: fbc-enabled
+          value: "$(tasks.read-config.results.fbc-enabled)"
+
     # Compute a hash of the content so pipeline logs can be uploaded with it
     - name: content-hash
       runAfter:
-        - read-config
+        - resolve-pr-type
       taskRef:
         name: content-hash
+      when:
```

Comment:
Some tasks depend on a result of this one. In case we skip it using `when` the follow-up task will be skipped.

Reply:
Thanks, fixed.